report_hm_9.md

```diff
@@ -0,0 +1,1521 @@
+```
+
+The connection is established,  Flags [P.] approves it. <br/>
+(Далее соединение в состоянии Established, Flags [P.] тому подтверждение.)
+
+## 3
+
+Flags [F.] is a sign of connection finishing. <br/>
+(Знаком завершения TCP является фрагмент с Flags [F.].)
+
+The client had inisilized the clousing of connection. The client and server are exchanging with last fragments. <br/>
+(Клиент инициировал завершение соединения. Клиент и серрвер обменияваются заключительными пакетами и подтвержениями их получения.)
+
+```
+192.168.33.1.60114 > 192.168.33.200.ssh: Flags [F.], cksum 0xd83e (correct), seq 4510, ack 3190, win 501, options [nop,nop,TS val 1016909714 ecr 4294829216], length 0
+21:26:40.563992 IP (tos 0x10, ttl 64, id 54459, offset 0, flags [DF], proto TCP (6), length 52)
+    192.168.33.200.ssh > 192.168.33.1.60114: Flags [.], cksum 0xc440 (incorrect -> 0xd8c6), seq 3190, ack 4511, win 363, options [nop,nop,TS val 4294829218 ecr 1016909714], length 0
+21:26:40.582547 IP (tos 0x10, ttl 64, id 54460, offset 0, flags [DF], proto TCP (6), length 52)
+    192.168.33.200.ssh > 192.168.33.1.60114: Flags [F.], cksum 0xc440 (incorrect -> 0xd8b3), seq 3190, ack 4511, win 363, options [nop,nop,TS val 4294829236 ecr 1016909714], length 0
+21:26:40.582804 IP (tos 0x10, ttl 64, id 0, offset 0, flags [DF], proto TCP (6), length 52)
+    192.168.33.1.60114 > 192.168.33.200.ssh: Flags [.], cksum 0xd816 (correct), seq 4511, ack 3191, win 501, options [nop,nop,TS val 1016909733 ecr 4294829236], length 0
+```
+
+## 4 
+
+
+Capturing of requests
+
+```
+[user@localhost ~]$ sudo tcpdump -ni ens33 port http and src host 192.168.0.17 -XX
+tcpdump: verbose output suppressed, use -v or -vv for full protocol decode
+listening on ens33, link-type EN10MB (Ethernet), capture size 262144 bytes
+22:36:14.930067 IP 192.168.0.17.38930 > 93.184.220.29.http: Flags [.], ack 788752898, win 241, options [nop,nop,TS val 4036288 ecr 1904324565], length 0
+	0x0000:  749d 7914 1f16 000c 291b f1b4 0800 4500  t.y.....).....E.
+	0x0010:  0034 f490 4000 4006 4ba4 c0a8 0011 5db8  .4..@.@.K.....].
+	0x0020:  dc1d 9812 0050 0c52 af19 2f03 6a02 8010  .....P.R../.j...
+	0x0030:  00f1 fab5 0000 0101 080a 003d 96c0 7181  ...........=..q.
+	0x0040:  afd5                                     ..
+22:36:18.770081 IP 192.168.0.17.49478 > 64.233.165.156.http: Flags [.], ack 3318564460, win 878, options [nop,nop,TS val 4040128 ecr 3211561325], length 0
+	0x0000:  749d 7914 1f16 000c 291b f1b4 0800 4500  t.y.....).....E.
+	0x0010:  0034 787c 4000 4006 1b09 c0a8 0011 40e9  .4x|@.@.......@.
+	0x0020:  a59c c146 0050 18aa 3adf c5cd 466c 8010  ...F.P..:...Fl..
+	0x0030:  036e a765 0000 0101 080a 003d a5c0 bf6c  .n.e.......=...l
+	0x0040:  896d                                     .m
+```
+
+Capturing of responses
+
+<details><summary>$ sudo tcpdump -ni ens33 port http and dst host 192.168.0.17 -XX</summary>
```

Comment:
1. What if port 80 (http) is reconfigured on the destination side, and it is not more uses http protocol?
2. Can we capture requests and responses by the same command?